andela/activo-api
# #158975213 Correct validation error for AssetCategory post

#### What does this PR do?
Corrects the validation error for AssetCategory Post

#### Description of Task to be completed?
 * Update the asset_category mock data for the test.
 * Refactor failing asset_catory test.
 * Add an extra test case

#### How should this be manually tested?
* Run `git fetch origin bg-incorrect-validationerror-assetcategory-post-158975213:bg-incorrect-validationerror-assetcategory-post-158975213`, then `git checkout bg-incorrect-validationerror-assetcategory-post-158975213`
* Run the command `flask run`, to start the application.
*Send a post request to the url: `http://localhost:5000/asset-categories` with an input control that is not one of ( 'dropdown', 'checkbox', 'radio button', 'textarea', 'text’). You should receive an error message as shown in the screenshot below.

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
[158975213](https://www.pivotaltracker.com/story/show/158975213)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/24465059/43003035-e327aa8e-8c22-11e8-9f46-39fcf8642c73.png)



#### Questions:
N/A
